bash/scripts/get_latest_tag.sh

```diff
@@ -11,5 +13,12 @@ get-latest-tag() {
   else
     exit 1
   fi
+
+  # If tag already released, bail out
+  if curl -sf "https://versions.deis.com/v3/versions/stable/deis-${component}/${tag}"; then
```

Comment:
I think there's a flag that'll show the error when it actually fails. `-S`? So `curl -sSf`